Fabric.Authorization.Persistence.SqlServer/Fabric.Authorization.Persistence.SqlServer.csproj

```diff
@@ -26,7 +26,6 @@
   <ItemGroup>
     <Folder Include="Configuration\" />
     <Folder Include="DependencyInjection\" />
-    <Folder Include="Mappers\" />
```

Comment:
was this intentional?

Reply:
No - VS removed it, not sure why. Not all of our folders are marked as include.